tests/__main__.py

```diff
@@ -16,6 +16,7 @@
 from tests.test_linear_model import main as linear_main
 from tests.test_composite import main as composite_main
 from .test_serialization import main as serialization_main
+from .test_ensemble import main as ensemble_main
```

Comment:
Reminder to use absolute paths rather than relative paths.

Reply:
^ agreed.